lib/rules/prettier.js

```diff
@@ -21,6 +21,23 @@ export default class Prettier extends Rule {
     this.filePath = options.filePath;
   }
 
+  getPrettierOptions() {
```

Comment:
Let's add a jsdoc comment to this function with some description/context, as this function is kind of important now that a third-party consumer is planning to override it.